src/v2/Components/FollowButton/FollowArtistButton.tsx

```diff
@@ -155,6 +152,12 @@ export class FollowArtistButton extends React.Component<Props, State> {
           .getLinkedRecord("counts")
           .setValue(newFollowCount, "follows")
       },
+      variables: {
+        input: {
+          artistID: artist.internalID,
+          unfollow: artist.is_followed,
+        },
+      },
```

Comment:
When did prettier start alphabetizing object keys? (I don't mind, just seems new this week)

Reply:
Yep this week. It's an eslint autofixer: https://github.com/artsy/force/pull/6655

Only spot it drives me crazy is in the routes file. I kinda need `path` to be on top... but that's just fighting a losing battle